### DISCUSSION TITLE
can't find properties of "Remind"

### DISCUSSION THREAD

**Discussion by aqiuxx** [ramnes/notion-sdk-py#186] [Q&A] [answered]
i want to get the full information of datetime in database, it should contain all attributes:  `start time`, `end time` , `Remind`. 

but when i debug it in vscode, i can't find the attribute `Remind`,  as show below

<img width="882" alt="datetime" src="https://user-images.githubusercontent.com/19370351/229063231-e7f732a3-4f12-46a1-900f-ae1305323069.png">

**demo code**
```
notion = Client(auth=TOKEN_V2)
my_database = notion.databases.query(database_id)
for n in my_database['results']:
    date_start = n['properties']['trigger-time']['date']['start']
    date_end = n['properties']['trigger-time']['date']['end']
```

how can i get the attribute "Remind"

**Comment by ramnes** [comment, marked as answer]
Notion's API does not expose reminders currently.
